content/awell-orchestration/api-reference/queries/get-pathways.mdx

```diff
@@ -86,7 +86,7 @@ When specifying multiple filters, the resulting criteria is the conjunction of a
 
 ### Pagination and sorting
 
-Pagination and sorting are highly recommended. Not using pagination means that the API will return all pathways at once which will cause performance issues.
+Pagination and sorting are required for this query. If no pagination values are supplied, the API will return only the first 10 care flows, or 10 care flows from any specified offset.
```

Comment:
[C] We need to make sure this is listed as a breaking change in the corresponding changelog. Can you create a placeholder for the corresponding release page in the changelog as well as part of this MR?